_posts/2023-03-13-introducing-security-analytics.md

```diff
@@ -0,0 +1,72 @@
+
+Although unfortunate, occurrences like these now appear frequently in the media and have become almost commonplace in the modern world. With an ever-increasing volume of business data shared online and stored in the cloud, malicious attempts to disrupt companies’ business interests are now simply part of the reality of doing business. And in light of these threats, it’s therefore essential that companies have tools in place to detect and identify the signatures of possible attacks in real time so they can act to intervene as quickly as possible. To address these needs, OpenSearch now includes a package of pre-built features and functionality that allow businesses to quickly and easily roll out a security solution for confronting a full range of cyber threats. Security Analytics is this solution for OpenSearch, and it installs automatically with any distribution.
+
+## The role of Security Analytics in OpenSearch
+
+OpenSearch already includes powerful features that allow administrators to track and analyze activity within the cluster. They provide users the ability to identify anomalous behavior in system data and set up alerts and notifications to signal users when these events occur. However, scaling these existing tools to serve the specific needs of various industries, handling the volume of data they possess, and then assembling a team of professionals to manage and interpret the results, pose a number of challenges.
+
+Security Analytics takes a different approach. It lets businesses use their existing OpenSearch cluster for detecting security threats, rather than requiring them to bear the cost of duplicating and storing their data for a separate off-the-shelf security solution. Security Analytics also leverages a schema of rules that it applies to log data to look for matches with abnormal events. These standardized rules address specific threats known throughout the industry and form a knowledge base of known attacks. Combined with a number of robust features, the rules also allow Security Analytics to be highly customizable.
+
+## The parts of the whole
+
+To see how this works, let’s first look at some of the components in Security Analytics.
+
+### Detectors
+
+Central to Security Analytics are its detectors. Detectors are components that interpret log data, compare that data to a schema of rules, and define the triggers that generate findings and the alerts that notify administrators of possible security events in the system. Their advantage is that they offer flexibility through their configuration, and this allows a business to fine tune its detection capabilities for specific threats, wider objectives, or targets in between.
```

Comment:
"in the system" -> "in your organization" or "in your infrastructure"
" and define the triggers" -> "enable you to configure alert triggers"